pkg/sense_hat_adapter.py

```diff
@@ -144,7 +144,6 @@ def update(self):
         elif self.name == 'temperature':
             value = self.device.controller.temperature
         else:
-            print("warning: %s not handled" % self.name)
```

Comment:
why remove this line ?

Reply:
Just was adding lots of stuff in the log.  But up to you.  I don't really understand, but the logging sometimes works and sometimes doesn't.